api/nodes.go

```diff
@@ -515,6 +515,8 @@ type DriverInfo struct {
 
 // HostVolumeInfo is used to return metadata about a given HostVolume.
 type HostVolumeInfo struct {
+	//ID       string
+	//Name     string
```

Comment:
I think this is what gets returned by the [Read Node API](https://developer.hashicorp.com/nomad/api-docs/nodes#read-node), but it's a map by name. Adding the ID makes sense but the name would be redundant.